packages/react/vite.config.ts

```diff
@@ -2,10 +2,15 @@ import react from '@vitejs/plugin-react';
 import path from 'node:path';
 import { defineConfig } from 'vite';
 import dts from 'vite-plugin-dts';
+// import mkcert from 'vite-plugin-mkcert';
 
 export default defineConfig({
     server: {
         port: 3001,
+        https: {
```

Comment:
@ashutoshdhande Please remove the usage keys & certs in this PR including this file.